src/appleseed/renderer/kernel/lighting/lighttree.cpp

```diff
@@ -247,8 +247,8 @@ float LightTree::recursive_node_update(size_t node_index, size_t node_level)
 }
 
 std::pair<size_t, float> LightTree::sample(
-    const foundation::Vector3d    surface_point,
-    float                         s) const
+    const foundation::Vector3d&     surface_point,
+    float                           s) const
```

Comment:
One minor thing I don't think I mentioned: whenver we indent stuff, we try to stick to tab "stops", i.e. multiples of 4 spaces.